src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Button, EntityHeader, Flex, Sans } from "@artsy/palette"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { TouchableHighlight, TouchableWithoutFeedback } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface FeaturedArtistsProps {
+  featuredArtists: any
```

Comment:
I think this should be typed as `FeaturedArtists_featuredArtists` instead of `any`, right? You can import it from the `__generated__` directory.

Reply:
Here's an example of that for reference! https://github.com/artsy/emission/blob/3adc3a7803fe5091bef60577eed9869ec26c1c31/src/lib/Scenes/Collection/Components/CollectionHeader.tsx#L2